.travis.yml

```diff
@@ -24,20 +26,72 @@ jobs:
           name: Lint /examples
           script: npm run lint examples
 
-        - stage: build-libs
+        - stage: build-components-lib
           name: Build /components
           script: npm run build:components
-        - stage: build-libs
+
+        - stage: build-doc-lib
           name: Build /doc-lib
           script: npm run build:doc-lib
 
-        - stage: build-app
+        - stage: build-app-aot
           name: Build /examples
-          script: npm run build:examples
+          before_script:
+              - cd ./projects/doc-lib
+              - npm install
+              - npm link
```

Comment:
We shouldn't need to npm link. I've mentioned it's black magic, and you're not running the two steps that its documentation says are required to get a link working